### PR TITLE
Add fake users to seed (closes #234)

### LINKED ISSUE #234: Create Sample User Data
AS a logged in User
I WANT TO see some sample users who are already in the system per default
TO play with the system and try out its functionality

ACCEPTANCE CRITERIA
- [ ] Every time the system is refreshed I want to have 10 default users configured per script
      - [ ] They should have normal realistic names and attributes filled out
           - First Name
           - Last Name
           - EMail 
           - Birthday
           - Telefon Number
           - Favorite Sports
           - If no big overhead a profile picture
(If to difficult to come up with people than use http://de.fakenamegenerator.com/gen-random-gr-gr.php)

### PR DESCRIPTION
**Pull request by nikriek**
Connects #234

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -101,6 +101,8 @@ gem 'awesome_nested_fields'
 # A modern, responsive front-end-framework based on Material Design
 gem 'materialize-sass'
 
+gem 'faker'
```
Comment (Askir) [inline Gemfile]: please add a short description of what this gem does and preferably a repository link for further documentation

File: db/seeds.rb [hunk 2]
```diff
@@ -55,6 +56,24 @@
...
+    last_name: Faker::Name.unique.last_name,
+    email: Faker::Internet.unique.email,
+    password: 'seed1234',
+    password_confirmation: 'seed1234',
+    birthday: Faker::Date.birthday(18, 65),
+    telegram_username: 'eb',
+    telephone_number: '12345654321',
+    avatar: File.open("#{Rails.root}/db/seed_data/avatar3.png"),
+    favourite_sports: sports[rand(sports.length), rand(sports.length - 1) + 1].join(', ')).save(validate: false)
+end
```
Comment (Askir) [inline db/seeds.rb:75]: It's not really the job of the seeds file to add example data. The seeds file should only specify the minimal data that is needed for operating the system at all.

But honestly I think for now this is fine. Just consider opening a ticket to extract this behaviour into a sperate rake task. We also have another ticket for even more example data that you could link to: #224

File: Gemfile (hunk 1 above)
Comment (nikriek) [reply Gemfile]: Done

File: db/seeds.rb (hunk 2 above)
Comment (nikriek) [reply db/seeds.rb:75]: Jep, you are right. I added a comment to the ticket that you mentioned.

**Review by Lariiii — APPROVED** [review]

**Pull request merged**